### ISSUE TITLE
ContentChildren get different children when add formArrayName

### ISSUE THREAD

**Issue by vthinkxie** [angular/angular#27504]
# 🐞 bug report

### Affected Package

@angular/forms

### Is this a regression?

No

### Description

when add `formArrayName` to the ng-container, the child component could not be accessed by the parent component

## 🔬 Minimal Reproduction

https://stackblitz.com/edit/angular-2vcmz3?file=src%2Fapp%2Fapp.component.html

## 🔥 Exception or Error
<pre><code>
ContentChildren get nothing when add `formArrayName`
should have same behavior
</code></pre>

## 🌍  Your Environment

**Angular Version:** 7.1

**Anything else relevant?**

**Comment by alexzuza** [comment]
You can set `descendants` option to true in order to query non-direct children:
```
@ContentChildren(HelloChildComponent, {descendants: true}) 
```

https://stackblitz.com/edit/angular-94aav9?file=src/app/hello.component.ts

Also check this thread https://github.com/angular/angular/issues/14769#issuecomment-356609267

**Comment by vthinkxie** [comment]
@alexzuza 
Yes, but I don't want to get non-direct children because the component may be used in a nested way.
The behavior should be consistent whether or not `formArrayName` are added
